gobra.go

```diff
@@ -28,16 +28,14 @@ package gobra
 
 import (
 	"bytes"
-	"fmt"
 	"html/template"
-
-	"honnef.co/go/js/dom"
-
-	"github.com/go-humble/view"
 )
 
 // Command holds information about a cobra command
 type Command struct {
+	// is the command toplevel or not
```

Comment:
Usually we want the comment to be a full sentence and start with something like "Toplevel specifies ...". I do concede, however, that the comment directly below this doesn't exactly follow this best practice, either.